.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "doorman"]
```

Comment:
Just wondering: why is this a submodule and not a hex dependency?

Reply:
I was following their documentation in their git readme, which matched the status of their repo but not their published package. This unblocked me, but I might get rid of the code that requires the newer version, or switch once they publish

Reply:
Though now reading through the hex docs it looks like I can just reference a GitHub there. Will change that